metrics/heapster.go

```diff
@@ -121,14 +121,13 @@ func createAndRunAPIServer(opt *options.HeapsterRunOptions, metricSink *metricsi
 		glog.Errorf("Could not create the API server: %v", err)
 		return
 	}
-	healthz.InstallHandler(server.Mux, healthzChecker(metricSink))
```

Comment:
> There's a way to add health checks. Did you accidentally lose this entirely?

Whoops, yeah, looks like this was installing the HTTP handler with a custom checker.  I'll add the custom checker back in